packages/workspace/src/browser/workspace-frontend-contribution.ts

```diff
@@ -417,12 +417,18 @@ export class WorkspaceFrontendContribution implements CommandContribution, Keybi
             }
         } while (selected && exist && !overwrite);
         if (selected) {
+            const temp = new URI(uri.toString() + '(copy).tmp');
```

Comment:
The variable `temp` is not used outside of the `try` block, therefore I think it should be placed inside.

Reply:
Ok. I moved it inside.